src/main/main.ts

```diff
@@ -93,6 +96,7 @@ ipcMain.on('user-quit', () => {
 app
   .whenReady()
   .then(async () => {
+    await AppDataSource.initialize();
```

Comment:
Should we retry or add something like an smoke test to ensure that if it fails the first time, we try to recover from it?

Reply:
I find smoke testing useful when you can recover from the smoke test failing case. This is not the case, if we fail to run this promise, we are sold and the app won't work properly, so I think a retry or an smoke test won't help us here. This method initializes your data source, runs schema migrations if needed and prepares the db file in case it doesn't exists